src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/HandEmbedProjection.java

```diff
@@ -11,4 +11,6 @@ public interface HandEmbedProjection {
   Integer getAnnouncedScore();
 
   Collection<CardEmbedProjection> getCards();
+
+  ParticipationEmbedProjection getParticipation();
```

Comment:
Wenn wir grad bei den Feldern sind: ich habe mir überlegt, ob es sinnvoll wäre, z.B. auf der Hand ein activ-Flag zu haben, damit wir mit wenig Aufwand die aktuelle Hand eines Spielers erwischen. Also bei den vergangenen "Händen" würde man dieses Flag dann ablöschen. Oder aber wir löschen gleich die ganzen Hände, wenn wir sie nicht mehr brauchen?

Reply:
Die Hände bleiben würde ich sagen, und ja das wäre praktisch, wenn man einfach wüsste, ob man dran ist.
Aber nicht in diesem PR

Reply:
ok, dann mach ich das mit meinem Issue